### DISCUSSION TITLE
What about adding Bluesky?

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#999] [Ask a question] [answered]
I am not sure if this is the right place to ask for a plugin...

The social network called "Blue Sky" is growing very fast, now it has more than 10 milion users.

It would be great to have a plugin for the Yellow CMS (like a comment system or a button to share a blog post on Blue Sky).

(I can't create one because I really don't have all the knoledge that is required for this...)

There is a list showing many tools for the blue sky here:

[https://docs.bsky.app/showcase](https://docs.bsky.app/showcase)

I intend to use this social network more often from now on.

This is just a request, I am not waiting for a reply ; )

Thank you.

**Comment by GiovanniSalmeri** [comment, marked as answer]
> It would be great to have a plugin for the Yellow CMS (like a comment system or a button to share a blog post on Blue Sky).

Hello! I have updated [Minishare](https://github.com/GiovanniSalmeri/yellow-minishare), now it includes Bluesky. Not tested, but it should work 🦋

(Could you edit the title of your question so that it is in the form of a question? For example, "What about adding Bluesky?" or something similar. Thank you!)

**Reply by ghost** [reply]
Hi @GiovanniSalmeri !

I will install your plugin.

Thank you for letting us know.

**Reply by ghost** [reply]
Hi again!

I installed your plugin, and yes, it works with bluesky too. Perfect, thank you!
